index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <section class="status-bar">
+        <img class="time" src="img\Time Style.svg" alt="time">
+        <img class="network-battery" src="img\Group 3.svg" alt="network and battery">
+    </section>
```

Comment:
- I feel this part of the website does not need to be included, I was only used in the Figma template to indicate a mobile design. I'd suggest you remove it 😄 😄 

Your project:
<img width="284" alt="mobile top" src="https://user-images.githubusercontent.com/73607512/198711548-5051a6cb-6213-48d6-9a00-27a671f497a9.PNG">